docs/tutorials/proposals/network-parameter-proposal.md

```diff
@@ -44,6 +44,26 @@ The contents of a `changes` object specifies what will be different after the pr
 
 **Value** is the new value you're proposing that the network parameter should have.
 
+Some network parameters include multiple pieces of information in one value. Changes to one of those network parameters needs to include the correct formatting. You can see how the existing value is formatted on [Explorer ↗](https://explorer.vega.xyz/network-parameters).
+
+For example, a proposal to change the `rewards.activityStreak.benefitTiers` network parameter would look like:
+
+```json title="Code sample for multiple values"
+"value": 
+" { \"tiers\": [ 
+    { \"minimum_activity_streak\": 1, 
+      \"reward_multiplier\": \"1.05\", 
+      \"vesting_multiplier\": \"1.05\" }, 
+    { \"minimum_activity_streak\": 2, 
+      \"reward_multiplier\": \"1.10\", 
+      \"vesting_multiplier\": \"1.10\" }, 
+    { \"minimum_activity_streak\": 3, 
+      \"reward_multiplier\": \"1.20\", 
+      \"vesting_multiplier\": \"1.20\" } 
+  ]
+}"
```

Comment:
**nitpick (blocking)**: Annoyingly what you've got here is JSON-with-escape-characters, rather than JSON. While the example is valid for pasting in to a terminal or similar, it's not actually valid JSON. To make it correct, you need to remove all the `\`s.